lib/api/src/lib/stories.ts

```diff
@@ -117,14 +117,6 @@ const warnChangedDefaultHierarchySeparators = deprecate(
   `
 );
 
-const toKey = (input: string) =>
-  input.replace(/[^a-z0-9]+([a-z0-9])/gi, (...params) => params[1].toUpperCase());
-
-const toGroup = (name: string) => ({
-  name,
-  id: toKey(name),
-});
```

Comment:
`toGroup` only adds an `id` prop, but this prop was never really used because it got overwritten by the `id` created through `sanitize`. So we don't need this mapping at all.